### DISCUSSION TITLE
2021-12-01 Weekly Meeting 32 Agenda / Notes

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#132] [Weekly Meetings]
Weekly Meeting 32 is happening on Wednesday December 1st, 2021 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Agenda

   - Welcome new members and attendance
   - Backlog refinement
     - Github Milestones is a much better way of swarming around sub projects (brief demo)
     - [VKet Booth ](https://github.com/omigroup/omigroup/issues/89)
     - [Continue discussion about partner program suggested by Thiizic of Futurist Foundation](https://github.com/omigroup/omigroup/issues/121)
     - https://github.com/omigroup/omigroup/issues/109
     - [website](https://github.com/omigroup/website/projects/1)
   - [glTF subgroup](https://github.com/omigroup/gltf-extensions/discussions/31)
     - Khronos Liaison update? 

 - Questions / Decisions:    
   - https://github.com/omigroup/omigroup/issues/129
   - Do we want to create a working group for OMI-Chan? 
   - How about using roles that match up with working groups?
   - [What does the fiscal development group need?](https://github.com/omigroup/omigroup/discussions/67#discussioncomment-1719482) - does the core omigroup repo work for managing this or do we want a repo for the working group? See notes. 
   - Do we have any new CFP submissions? 
     - https://github.com/omigroup/omigroup/issues/134

 ## Add Ons
   - Upcoming Events: 
     - WebXR Hackathon - "Immersive Web Fest"
     - [Avacon](https://www.avacon.org/blog/)
     - XR Safety Week  [Reminder Call for participation: XR Safety Week](https://github.com/omigroup/omigroup/issues/124) follow up reminder 
    - Reminder Twitter Spaces with Evo / Jin
    - others?

**Comment by madjin** [comment]
Open metaverse community calendar: https://github.com/omigroup/omigroup/issues/130

**Comment by robertlong** [comment]
WASM UGC Scripting Working Group: https://github.com/omigroup/omigroup/issues/133

**Comment by mrmetaverse** [comment]
Robert has some bandwidth to work on site on Wednesdays

**Comment by mrmetaverse** [comment]
update on khronos liason: working through the process now. Probably robert and fire representing the working group. But still steps to go through. For the official liason group, OMI is not a registered corp, so Robert can rep OMI as liason but will do so using his Matrix credentials. Called this out for transparency.

**Comment by mrmetaverse** [comment]
OMI-Chan Avatar effort

**Reply by mrmetaverse** [reply]
vote to form working group https://github.com/omigroup/omigroup/issues/135

**Comment by mrmetaverse** [comment]
Make roles that match up with working groups

**Reply by mrmetaverse** [reply]
https://github.com/omigroup/omigroup/issues/136

**Comment by mrmetaverse** [comment]
not sure if we recorded this session, but I forgot to take attendance
